db/db_impl/db_impl.cc

```diff
@@ -1496,6 +1496,29 @@ bool DBImpl::SetPreserveDeletesSequenceNumber(SequenceNumber seqnum) {
   }
 }
 
+Status DBImpl::IncreaseFullHistoryTsLow(ColumnFamilyHandle* column_family,
+                                        std::string& ts_low) {
+  if (column_family == nullptr) {
+    return Status::InvalidArgument("column family handle is null");
+  }
+  auto cfh = static_cast_with_check<ColumnFamilyHandleImpl>(column_family);
+  return IncreaseFullHistoryTsLow(cfh->cfd(), ts_low);
+}
+
+Status DBImpl::GetFullHistoryTsLow(ColumnFamilyHandle* column_family,
+                                   std::string* ts_low) {
```

Comment:
Can `ts_low` be null? It does not seem to make much sense to call this method but not specify where to return. Either add a check, or change its type to `std::string&`.

Reply:
fixed,